CMakeLists.txt

```diff
@@ -94,6 +94,13 @@ set(IREE_ALL_HAL_DRIVERS
 if( IREE_HAL_DRIVERS_TO_BUILD STREQUAL "all" )
   set( IREE_HAL_DRIVERS_TO_BUILD ${IREE_ALL_HAL_DRIVERS} )
 endif()
+
+# For cross compilation towords Android, we don't want LLVM JIT HAL driver.
+if(ANDROID AND LLVM IN_LIST IREE_HAL_DRIVERS_TO_BUILD)
```

Comment:
I kind of don't like this implicit logic. If someone wants to try to build the LLVM JIT for Android then they should feel free to. Seems we can just not specify it in DRIVERS_TO_BUILD? Maybe the thing to do is change the *default* if this is Android.

Reply:
I don't quite see how they are gonna use it though. It's broken and will not be supported anyway.. But i'm okay changing the default too.

Reply:
Well it's just kind of the principle of the thing. Like if someone wants to build our project with gcc, we shouldn't make that impossible just because it's broken. We should make the default behavior something that works, but if I set `-DIREE_HAL_DRIVERS_TO_BUILD=LLVM -DANDROID` to see what errors there are building that driver for android then I'm going to be confused if everything just passes and it doesn't build that driver.